WordPress/gutenberg
# NUX: Make tips collapsed by default

Closes #9242. Fixes #12358.

![nux](https://user-images.githubusercontent.com/612155/48600251-44f91500-e9bf-11e8-90fe-9b417a595904.gif)

Makes the NUX guide less overwhelming by collapsing the first tip in the guide.

What's changed:

- `DotTip` has a new `isCollapsible` prop which marks it as being a collapsible tip
  - Collapsible tips are opened and closed by clicking on the indicator
  - I've increased the size of the indicator so that it's easier to click
  - The first tip is marked as `isCollapsible`
- Dismissing and disabling tips works as before

#### How to test

1. Clear out your local storage or re-enable tips
2. Note that the first tip is now a collapsed pulsating dot
3. Click on it to open up the first tip
4. Click through the rest of the tips
